examples/sm/run.sh

```diff
@@ -1 +1 @@
-./bin/sm_main_9.0_x86_64 market  ../../dataset/small/tree.mtx --pattern-graph-type=market --pattern-graph-file=../../dataset/small/query_sm.mtx --undirected=1 --pattern-undirected=1 --num-runs=1 --quiet=False
+./bin/sm_main_10.0_x86_64 market  ../../dataset/small/tree.mtx --pattern-graph-type=market --pattern-graph-file=../../dataset/small/query_sm.mtx --undirected=1 --pattern-undirected=1 --num-runs=1 --quiet=False
```

Comment:
Can you add more than just one small example in the `run.sh` file? And use other examples to make sure we follow some sort of standard for this, for example, you hard code the binary name (which will change depending on what CUDA version you use) and some of the other files actually search for the latest binaries.

Reply:
Sure, that makes sense.